servicenow/connector-servicenow.lua

```diff
@@ -1,275 +1,920 @@
 #!/usr/bin/lua
---------------------------------------------------------------------------------
--- Centreon Broker Servicenow Connector
---
---------------------------------------------------------------------------------
-
-local curl = require "cURL"
-
-local serviceNow
-
--- Class for Service now connection
-local ServiceNow = {}
-ServiceNow.__index = ServiceNow
-
-function ServiceNow:new(instance, username, password, clientId, clientPassword)
-  local serviceNow = {}
-  setmetatable(serviceNow, ServiceNow)
-  serviceNow.instance = instance
-  serviceNow.username = username
-  serviceNow.password = password
-  serviceNow.clientId = clientId
-  serviceNow.clientPassword = clientPassword
-  serviceNow.tokens = {}
-  serviceNow.tokens.authToken = nil
-  serviceNow.tokens.refreshToken = nil
-  return serviceNow
-end
-
-function ServiceNow:getAuthToken ()
-  if not self:refreshTokenIsValid() then
-    self:authToken()
+
+  --------------------------------------------------------------------------------
+  -- Centreon Broker Service Now connector
+  --------------------------------------------------------------------------------
+
+  --------------------------------------------------------------------------------
+  -- Prerequisites:
+  --
+  -- You need a ServicecNow instance with the event manager module
+  -- You need your Oauth credentials
+  --
+  -- The lua-curl is required by this script:
+  -- yum install lua-curl epel-release
+  -- yum install luarocks
+  --------------------------------------------------------------------------------
+
+  --------------------------------------------------------------------------------
+  -- Parameters:
+  -- [MANDATORY] instance: your ServiceNow instance name
+  -- [MANDATORY] username: your OAuth user
+  -- [MANDATORY] password: your OAuth password
+  -- [MANDATORY] client_id: your OAuth client ID
+  -- [MANDATORY] client_secret: you OAuth client secret
+  -- [RECOMMENDED] log_level: level of verbose. Default is 1,
+  -- [OPTIONAL] http_proxy_string: default empty
+  --
+  --------------------------------------------------------------------------------
```

Comment:
Could you please remove this part? We will no longer include such details as comments.

Maybe dropping it in a README and including a link to the official Centreon documentation would be a better option nope?